vcr/vcr.go

```diff
@@ -565,7 +584,8 @@ func (c *vcr) Get(conceptName string, subject string) (concept.Concept, error) {
 
 	q.AddClause(concept.Eq(concept.SubjectField, subject))
 
-	vcs, err := c.Search(q)
+	// finding a VC that backs a concept is a runtime call, eg: usage is now
+	vcs, err := c.Search(q, nil)
```

Comment:
shouldn't `at` be `time.Now()` in that case?

Reply:
I'll adjust the description. At the lowest level, time.Now is used.